plenum/config.py

```diff
@@ -137,6 +137,11 @@
 LatencyWindowSize = 30
 LatencyGraphDuration = 240
 
+# This parameter defines minimal count of accumulated latencies for each client
+MIN_LATENCY_COUNT = 50
```

Comment:
I think requiring 50 **for each client** is too strict. Should we required 50 for all clients?